### PR TITLE
Dm 2048

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2048

## Description - what does this code do?
Adds new contact section to practice editor and practice view.

## Testing done - how did you test it/steps on how can another person can test it 
1. Log in as an admin.
2. Browse to any practice's `Contact` section of the practice editor.
3. Make sure the `Main email address` field is required before saving.
4. Add a main email address.
5. Add. remove, and edit cc'd email addresses as you see fit.
6. Browse to the show page of the same practice.
7. Scroll down to the contact section and confirm that the main email address is displayed as a `mail_to` link.
8. Click on the link and confirm that the cc'd email address(s) you entered are displayed correctly in the email window.
9. Make sure the style changes to the comments match the ones in the Figma designs.

## Screenshots, Gifs, Videos from application (if applicable)
Screen Shot 2020-09-13 at 8 46 57 PM
Screen Shot 2020-09-13 at 8 39 10 PM
Screen Shot 2020-09-13 at 8 46 35 PM

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/FogT5mBUVl8rE7OgWDQWeJ/VADM?node-id=276%3A0

## Acceptance criteria
- Email address displays on page and it's a "Mail To" link
- Hyperlink within text, no mailer form
- Practice owner can enter email address in blank text box
- Existing email addresses are carried over
- Copy: “Email [Practice Owner email address] with questions about this practice.”
- Comments NO mobile view just style makeover

## Definition of done
- [X] Unit tests written (if applicable)
- [X] e2e/accessibility tests written (if applicable)
- [X] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/views/commontator/comments/create.js.erb [hunk 1]
```diff
@@ -5,15 +5,18 @@ var reloadWithTurbolinks = (function () {
       scrollPosition = [ window.pageXOffset, window.pageYOffset ];
       Turbolinks.visit(window.location.toString(), { action: 'replace' });
   }
+  $(document).off('turbolinks:load', somethingAwesome);
 
-  document.addEventListener('turbolinks:load', function () {
-    $('.timeago').timeago();
-    $('#dm-practice-nav').sticky( {topSpacing: 20, zIndex: 10000, bottomSpacing: 200});
+  function somethingAwesome() {
```
Comment (ahay-agile6) [inline app/views/commontator/comments/create.js.erb:10]: lol, can we rename this? :)

File: app/assets/stylesheets/dm/variables/_colors.scss [hunk 2]
```diff
@@ -10,8 +10,10 @@ $dm-color-blue-50v: #0076d6;
 $dm-color-blue-70v: #0b4778;
 $dm-color-blue-80v: #112f4e;
 $dm-color-red-50v: #e52207;
+$dm-color-red-alt: #d83933;
```
Comment (ahay-agile6) [inline app/assets/stylesheets/dm/variables/_colors.scss:13]: is this the name of the color in uswds?

File: app/views/practices/show/authenticated/_show_authenticated.html.erb [hunk 3]
```diff
@@ -309,17 +307,17 @@
...
-        <div class="desktop:grid-col-9 grid-col-12 p0">
-          <div class="contact-section practice-section">
-            <%= render partial: 'support_team' %>
-          </div>
-        </div>
-      </div>
-    </div>
-  </section>
-<% end %>
+<%# if @practice.va_employees.any? || @practice.support_network_email.present? %>
```
Comment (ahay-agile6) [inline app/views/practices/show/authenticated/_show_authenticated.html.erb:310]: can we delete this code if we're not using it?

**Review by ahay-agile6 — CHANGES_REQUESTED** [review]
looks good! just a couple of things

File: app/views/practices/show/authenticated/_show_authenticated.html.erb (hunk 3 above)
Comment (JoshingYou1) [reply app/views/practices/show/authenticated/_show_authenticated.html.erb:310]: removed :)

File: app/assets/stylesheets/dm/variables/_colors.scss (hunk 2 above)
Comment (JoshingYou1) [reply app/assets/stylesheets/dm/variables/_colors.scss:13]: Removed

File: app/views/commontator/comments/create.js.erb (hunk 1 above)
Comment (JoshingYou1) [reply app/views/commontator/comments/create.js.erb:10]: Hahaha, omg!! Of course!

**Review by emyl3 — APPROVED** [review]
Great work, Josh! This looked good to me and worked as intended! 👍

**Review by ahay-agile6 — APPROVED** [review]

**Pull request merged**